services-add-event.md

```diff
@@ -0,0 +1,77 @@
+
+![](images/events-get-started.png)
+
+The *Add events* dialog opens, providing a listing of events that can be filtered based on the products that are available to your organization.
+
+![](images/events-add.png)
+
+Choose the event that you would like to add and select **Next** to begin configuring the events.
+
+![](images/events-add-select.png)
+
+## Configure
+
+The first step in configuration is to choose your event subscriptions. This requires you to select from the provided list which events you would like to receive notifications about from your chosen event providers. As events are chosen, they appear under *Subscribed Events* in the left navigation.
+
+![](images/events-configure.png)
+
+## Credentials
+
+The next step in configuring your event registration is to configure the authentication credentials. This could be a service account (JWT) or OAuth, depending on the type of event being configured. The example in this document uses OAuth 2.0 authentication and authorization.
+
+> **Note:** To learn more about authentication and available types, visit the [authentication documentation](https://www.adobe.com/go/devs_authentication).
+
+To configure an Event using OAuth 2.0 authentication and authorization, you must first select the platform where you want to use this integration: Web, iOS, or Android. 
+
+Once you have selected a platform, you will be required to provide a *Redirect URI*, which is a fallback URI to be used if the authorization request contains a redirect URI which doesn't match the Redirect URI list or doesn't contain a `redirect_uri` parameter.
+
+> **Note:** The *Redirect URI* must use HTTPS (for example, `https://redirect.com/uri/etc`) and cannot be a regular expression.
+
+After entering a redirect that matches the appropriate formatting, you can then select **Next** to proceed with event configuration.
+
+![](images/events-configure-auth.png)
+
+## Registration details
+
+To complete the configuration process, you must provide *Event registration details* including a custom name and description to differentiate this event registration from others in the project.
+
+You must also register a webhook by providing a **Webhook URL** to receive notifications about events and specifying how often you would like to receive events. Each event results in an HTTP request to the webhook URL, notifying your application about events. You can select the **Delivery Style** you prefer, receiving one event at a time ("Single") or multiple events together ("Batch").
+
+To learn more about webhooks, visit the [webhooks documentation](https://www.adobe.com/go/devs_webhooks).
+
+After completing the *Event registration details* select **Save configured events** to complete the configuration.
+
+![](images/events-configure-registration.png)
+
```

Comment:
I realized that by going through the OAuth Events flow some of the options are missing. 
For Events that use JWT, the developer has multiple ways to consume the Events: optional Webhook, optional Runtime action (if Runtime enabled and actions exist) and Journaling. Journaling provides API access to events and the endpoint is displayed on the Event Registration details page. 
No rush - we can add that after launch

Reply:
I've noted this for Phase II and will update immediately after launch.